nilearn/_utils/bids.py

```diff
@@ -0,0 +1,63 @@
+"""BIDS related utilities."""
```

Comment:
wonder if this should not go instead in the interface/bids module

Reply:
I just looked at the interfaces module and BIDS isn't something to interface with, it's a data organization system, but, well I don't use it so what do I know.
But after glancing at some of the other functions there,  such as generate metadata, etc, you're suggestion night be the better place for this code.

Reply:
moved things

also for the reason that it centralizes all bids things in one place